neural_compressor/torch/algorithms/weight_only/__init__.py

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from .rtn import rtn_quantize
+from .rtn import RTNQuantizer
```

Comment:
This file has been emptied by https://github.com/intel/neural-compressor/pull/1742, please rebase the master.